cfgov/unprocessed/js/routes/on-demand/simple-chart/datetime-styles.js

```diff
@@ -1,45 +1,40 @@
 /* eslint complexity: ["error", 10] */
 import styles from './line-styles.js';
+const msYear = 365 * 24 * 60 * 60 * 1000;
 
 const datetime = {
   ...styles,
   chart: {
     ...styles.chart,
     spacingTop: 0,
     spacingBottom: 0,
-    marginTop: 70
+    marginTop: 100
   },
   xAxis: {
     ...styles.xAxis,
     type: 'datetime',
-    minRange: 30 * 24 * 3600 * 1000,
     startOnTick: true,
     labels: {
-      ...styles.xAxis.labels,
-      format: 'Jan<br/>{value:%Y}'
+      ...styles.xAxis.labels
```

Comment:
If IE11 support is needed, you might double-check transpiling of the spread operator in IE11. You may need to include a polyfill.

Reply:
The transpiling appears to work in my testing